pandas/core/categorical.py

```diff
@@ -118,6 +118,10 @@ def maybe_to_categorical(array):
 
 Assigning to `categories` is a inplace operation!
 
+Note
+----
+Categories getter returns all categories and not only those present in the data
```

Comment:
Should I rebase and force push to delete the commit that alters categories docstring?